rep-2000.rst

```diff
@@ -187,10 +190,13 @@ Significant errors discovered in Tier 1 platforms can impact release dates and w
 
 Tier 2
 
-Tier 2 platforms are subject to ad hoc testing and binary packages may not be provided or supported.
+Tier 2 platforms are subject to periodic CI testing which runs both test builds and unit tests with publically accessible results.
+The CI is expected to be run at least within a week of a relevant changes for the current state of the rosdistro.
+Binary packages may not be provided or supported.
```

Comment:
I would suggest to combine this new sentence with the next one with a clear preference for making an archive available:

> Package-level binary packages may not be provided but providing a downloadable archive of the built workspace is encouraged.